src/clef.ts

```diff
@@ -127,7 +126,7 @@ export class Clef extends StaveModifier {
       this.size = size;
     }
     this.clef.point = this.musicFont.lookupMetric(`clef.${this.size}.point`, 0);
-    this.glyph = new Glyph(this.clef.code, this.clef.point, {
+    this.glyph = new Glyph(this.clef.code, this.clef.point ?? 0, {
```

Comment:
Here `this.clef.point` must always be defined, otherwise it's a silent failure. So a good place to use `assertDefined` like my above comment.